apps/frontend/src/pages/profile/libs/components/delete-account/styles.module.css

```diff
@@ -18,3 +18,7 @@
 	font-size: 16px;
 	line-height: 150%;
 }
+
+.button-wrapper {
+	min-width: 190px;
```

Comment:
the same, on figma this button just takes it's content width, we need to use flex here to align item to one of the sides